user_tools/src/spark_rapids_tools/tools/qualx/preprocess.py

```diff
@@ -409,7 +413,7 @@ def combine_tables(table_name: str) -> pd.DataFrame:
 
     # job_map_tbl = combine_tables('job_map_tbl')
     job_stage_agg_tbl = combine_tables('job_stage_agg_tbl')
-    wholestage_tbl = combine_tables('wholestage_tbl')
+    whole_stage_tbl = combine_tables('wholestage_tbl')
```

Comment:
I rememeber there was something odd about job_stage-aggretaes.
When the tools split those tables in two different files, the QualX merged them back.
Are we still doing this? If so, why do we still do it?

- Another question about the feature_tables:  is `job_+_stage_level_aggregated_task_metrics ` in the table entry correct? I don't see any other reference to that literal string in the python code.
`(job_stage_agg_tbl, 'job_+_stage_level_aggregated_task_metrics')`

Reply:
Yes. `job` metrics are not used anywhere. We should be able to remove them and use only `stage` level metrics.

cc: @leewyang

Reply:
I believe this is correct (see comment on L945), but I think that can be in a followup PR.

Reply:
Created a tracking issue https://github.com/NVIDIA/spark-rapids-tools/issues/1234 to remove reading of job level aggregate metrics.